app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -0,0 +1,18 @@
+package ar.com.wolox.android.example.ui.home.news
+
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.FragmentNewsBinding
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import javax.inject.Inject
+
+class NewsFragment @Inject constructor() : WolmoFragment<FragmentNewsBinding, NewsPresenter>(), NewsView {
```

Comment:
Is this `@Inject constructor()` needed

Reply:
I think he will need it on later cards 🤔